tests/qir_test_files/WASM_noinputs.ll

```diff
@@ -0,0 +1,16 @@
+source_filename = "Generated from input pytket circuit and wasm_adder.wasm file."
+
+declare i32 @__quantum__hybrid__add_one__body(i32)
```

Comment:
If this is a no-inputs function shouldn't it be `()` rather than `()` in the QIR?

Reply:
Not sure. We explicitly made it such that if there isn't any input in the WASMOp, then the SSA variable input  in QIR Is initialised to 0.